sdk/metric/aggregator/maxsumcount/msc.go

```diff
@@ -86,7 +88,7 @@ func (c *Aggregator) Checkpoint(ctx context.Context, _ *export.Descriptor) {
 
 	c.checkpoint.count.SetUint64(c.current.count.SwapUint64Atomic(0))
 	c.checkpoint.sum = c.current.sum.SwapNumberAtomic(core.Number(0))
-	c.checkpoint.max = c.current.max.SwapNumberAtomic(core.Number(0))
+	c.checkpoint.max = c.current.max.SwapNumberAtomic(desc.NumberKind().Minimum())
```

Comment:
I like your approach here.  I'd like to do the same for minimum and maximum values, i.e.,

```
c.current.min.SwapNumberAtomic(desc.NumberKind().Maximum())
```

Reply:
nit: Maybe use `c.kind` instead of `desc.NumberKind().Minimum()`.